src/components/PageInfoCard/PageInfoCard.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { usePlayList } from 'context'
+import { useNavigate, useParams } from "react-router-dom"
+import { deletePlaylist } from 'utilities'
+import styles from "./PageInfoCard.module.css"
+
+const PageInfoCard = () => {
+    const navigate = useNavigate();
+    const { playlists, playListDispatch } = usePlayList();
+    const { playlistId } = useParams();
+    const data = playlists.find((playlist) => playlist._id === playlistId)
```

Comment:
if data is not used anywhere else, you can destructure videos here itself

Reply:
i am using title of data.